build.cake

```diff
@@ -766,12 +766,31 @@ Task("PackageZip")
         Zip(currentImageDir, File(ZIP_PACKAGE_SL), zipFiles);
     });
 
-Task("PackageMsi")
+
+Task("PackageRunnersMsi")
+    .IsDependentOn("CreateImage")
+    .WithCriteria(IsRunningOnWindows)
+    .Does(() =>
+    {
+        MSBuild("runners-msi/runners/nunit.wixproj", new MSBuildSettings()
+            .WithTarget("Rebuild")
+            .SetConfiguration(configuration)
+            .WithProperty("PackageVersion", packageVersion)
+            .WithProperty("DisplayVersion", version)
+            .WithProperty("OutDir", PACKAGE_DIR)
+            .WithProperty("InstallImage", IMAGE_DIR + "NUnit-" + packageVersion)
+            .SetMSBuildPlatform(MSBuildPlatform.x86)
+            .SetNodeReuse(false)
+        );
+    });
+
+
+Task("PackageFrameworkMsi")
```

Comment:
@rprouse's suggestion was to not distribute the framework msi, but keep the files around for one release in case we needed to put it in. Because of that I added the CI build, so we know if we broke it - but I'd be happy removing it entirely and resurrecting it if need be.

Let me know which you'd prefer. :-)